src/main/java/net/atos/zac/app/taken/TakenRESTService.java

```diff
@@ -204,6 +204,17 @@ public RESTTaak partialUpdateTaak(final RESTTaak restTaak) {
         return taakConverter.convertTaskInfo(updatedTask);
     }
```

Comment:
Zou je wat javadoc kunnen toevoegen wat het verschil is tussen partialUpdateTaak en updateTaak

Reply:
Toegevoegd